NEWS.md

```diff
@@ -1,11 +1,10 @@
-## 2.1.2 - 2021-06-17
+## 2.2.1 - 2021-06-23
```

Comment:
Why removing the 2.1.2 release?

Reply:
Because v2.1.2 is in the b2.1 branch only: https://github.com/folio-org/mod-login-saml/blob/b2.1/NEWS.md
This NEWS.md entry has incorrectly been ported from b2.1 branch to master branch.

Reply:
> Because v2.1.2 is in the b2.1 branch only: https://github.com/folio-org/mod-login-saml/blob/b2.1/NEWS.md
> This NEWS.md entry has incorrectly been ported from b2.1 branch to master branch.

Not true. The 2.1.2 fix was merged into master earlier. Besides, this is `NEWS.md `that should contain all releases. if go with your logic, this new 2.2.1 release should go to b2.2, not into master either. That does not make sense.

Reply:
v2.1.2 has been released from commit that is not in the repository:
https://github.com/folio-org/mod-login-saml/releases/tag/v2.1.2 = https://github.com/folio-org/mod-login-saml/commit/44cb806909f043d80e47294ded84ecf09117b7e2

The NEWS entry for 2.1.2 has been added to branch b2.1 first:
https://github.com/folio-org/mod-login-saml/pull/80/files

The NEWS entry for 2.1.2 has been copied to master in a second pull request:
https://github.com/folio-org/mod-login-saml/pull/81/files

https://github.com/folio-org/mod-login-saml/releases contains all releases order by release date.
NEWS.md file in master contains all releases ordered by release number.
Moving 2.1.2 to the position between 2.2.0 and 2.1.1 fixes the order.